share/spice/holiday/holidays.txt

```diff
@@ -0,0 +1,1367 @@
+17 may constitution day
+1848 revolution memorial day
+1956 revolution memorial day
+2016 municipal elections
+2016 primary elections
+228 memorial day
+228 memorial day observed
+25th of march
+2nd christmas day
+7 5 3 day
+80th anniversary of the constitution of the basque government in guernica
+abolition of slavery
+action day for tolerance and respect between people
+adelaide cup
+adhi esala full moon poya day
+administrative professionals day
+adwa victory day
+afghan victory day
+afl grand final friday
+africa day
+africa freedom day
+african union day
+agriculture and labor day
+aid el fitr
+aid el kebir
+air force birthday
+alaska day
+albanian alphabet day
+albanian community
+al hijra
+al isra wal miraj
+all saints day
+all saints eve
+all souls day
+ambedkar jayanti
+america day
+american eagle day
+american independence day
+american indian heritage day
+amtseinführung museveni
+amun jadid
+ancestors’ day
+annexation of guanacaste
+anniversary of khomeinis death
+anniversary of president laurent kabila’s assassination
+anniversary of prime minister patrice emery lumumba’s assassination
+anniversary of prime minister patrice emery lumumba’s assassination observed
+anniversary of the arengo
+anniversary of the decease of m r ŝtefánika
```

Comment:
if possible, we should also try to normalize UTF8 chars.

e.g. this would be `stefanika` -- I suspect a Perl module exists to do this. If not it's not very important